spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/options/SingleNodeOptions.java

```diff
@@ -63,6 +61,19 @@ public String getXD_TRANSPORT() {
 		return transport;
 	}
 
+	public void setPort(int httpPort) {
+		this.httpPort = httpPort;
+	}
+
+
+	public void setAnalytics(String analytics) {
+		this.analytics = analytics;
+	}
+
+	public void setTransport(String transport) {
+		this.transport = transport;
+	}
+
```

Comment:
Alternate setters required with boot 1.3, related to Relaxed binding. Not sure if we should remove the old ones @ericbottard